src/validation.cpp

```diff
@@ -1608,7 +1608,7 @@ PackageMempoolAcceptResult MemPoolAccept::AcceptMultipleTransactions(const std::
     }
 
     // Now that we've bounded the resulting possible ancestry count, check package for dust spends
-    if (m_pool.m_opts.require_standard) {
+    if (!args.m_bypass_limits && m_pool.m_opts.require_standard) {
```

Comment:
I think it's impossible for `m_bypass_limits` to be true in this function, so undoing this diff doesn't make a difference. Could we omit it or `Assume(!args.m_bypass_limits)`?

Reply:
Took feedback in other PR ahead of time https://github.com/bitcoin/bitcoin/pull/33504#discussion_r2389364292

I can definitely put an `Assume()` on it.